src/components/NurseryWithdrawals/WithdrawalLogRenderer.tsx

```diff
@@ -20,82 +20,93 @@ const useStyles = makeStyles((theme: Theme) => ({
-        listSeparator={strings.LIST_SEPARATOR}
-        moreSeparator={strings.TRUNCATED_TEXT_MORE_SEPARATOR}
-        moreText={strings.TRUNCATED_TEXT_MORE_LINK}
-      />
-    );
-  };
+    const createLinkToNurseryWithdrawalDetail = (iValue: React.ReactNode | unknown[]) => {
+      const nurseryWithdrawalDetailLocation = APP_PATHS.NURSERY_WITHDRAWALS_DETAILS.replace(
+        ':withdrawalId',
+        row.id.toString()
+      );
+      return (
+        <Link to={nurseryWithdrawalDetailLocation} className={classes.link}>
+          {iValue as React.ReactNode}
+        </Link>
+      );
+    };
 
-  if (column.key === 'speciesScientificNames') {
-    return <CellRenderer index={index} column={column} value={getTruncated(value)} row={row} />;
-  }
+    const getTruncated = (inputValues: any) => {
+      return (
+        <TextTruncated
+          stringList={inputValues}
+          maxLengthPx={COLUMN_WIDTH}
+          textStyle={{ fontSize: 14 }}
+          showAllStyle={{ padding: theme.spacing(2), fontSize: 14 }}
+          listSeparator={strings.LIST_SEPARATOR}
+          moreSeparator={strings.TRUNCATED_TEXT_MORE_SEPARATOR}
+          moreText={strings.TRUNCATED_TEXT_MORE_LINK}
+        />
+      );
+    };
 
-  if (column.key === 'plotNames' && value) {
-    return <CellRenderer index={index} column={column} value={getTruncated([value])} row={row} />;
-  }
+    if (column.key === 'speciesScientificNames') {
+      return <CellRenderer index={index} column={column} value={getTruncated(value)} row={row} />;
+    }
 
-  if (column.key === 'withdrawnDate') {
-    return <CellRenderer index={index} column={column} value={createLinkToNurseryWithdrawalDetail(value)} row={row} />;
-  }
+    if (column.key === 'plotNames' && value) {
+      return <CellRenderer index={index} column={column} value={getTruncated([value])} row={row} />;
+    }
 
-  if (column.key === 'hasReassignments') {
-    if (row.purpose === OUTPLANT && row.plotNames) {
+    if (column.key === 'withdrawnDate') {
       return (
-        <>
-          <CellRenderer
-            index={index}
-            column={column}
-            row={row}
-            value={
-              <Button
-                id='reassign'
-                label={strings.REASSIGN}
-                onClick={rowClick}
-                size='small'
-                priority='secondary'
-                className={classes.text}
-                disabled={isTrue(row.hasReassignments)}
-              />
-            }
-          />
-        </>
+        <CellRenderer index={index} column={column} value={createLinkToNurseryWithdrawalDetail(value)} row={row} />
       );
     }
-    return <CellRenderer index={index} column={column} row={row} value='' />;
-  }
 
-  return <CellRenderer {...props} />;
-}
+    if (column.key === 'totalWithdrawn') {
+      return <CellRenderer index={index} column={column} value={formatter.format(value as number)} row={row} />;
+    }
```

Comment:
this was the only code added here..everything else that's shown as 'modified' was not modified.. it's a PR diff artifact